Documentation/Diagnostics/PH2112.md

```diff
@@ -0,0 +1,44 @@
+# PH2112: Avoid overridde with new keyword
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2112 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidOverridingWithNewKeywordAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidOverridingWithNewKeywordAnalyzer.cs)
+| CodeFix  | Yes |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
```

Comment:
I don't think I would use the word "override" to describe this behavior. I would say something like "Hiding inherited members with the `new` keyword...